cmd/dapr.go

```diff
@@ -53,50 +58,52 @@ const (
 )
 
 var (
-	daprVer   daprVersion
-	logAsJSON bool
-	daprPath  string
+	cliVersion  string
+	versionFlag bool
+	daprVer     daprVersion
+	logAsJSON   bool
+	daprPath    string
 )
 
 // Execute adds all child commands to the root command.
 func Execute(version, apiVersion string) {
```

Comment:
this is the entry point for dapr CLI. `main.go` calls this Execute() method every time a dapr cli command is run.